packages/eslint-config/index.js

```diff
@@ -1,40 +1,42 @@
 module.exports = {
-  parser: '@typescript-eslint/parser',
+  parser: "@typescript-eslint/parser",
```

Comment:
Oh no, you changed the single quote to double 😅

Reply:
Not even sure how that happened, have everything set to single, but fixed it now.
I do see quite a lot of files still have double from before though.